src/components/frame/v5/pages/UserPreferencesPage/UserPreferencesPage.tsx

```diff
@@ -173,20 +173,25 @@ const UserPreferencesPage: FC<UserPreferencesPageProps> = ({
         {/*   /> */}
         {/*   <Switch id="disable-notifications" /> */}
         {/* </div> */}
-        {/* <span className="divider" /> */}
-        {/* <h5 className="heading-5"> */}
-        {/*   {formatText({ */}
-        {/*     id: 'userPreferencesPage.displayPreferences', */}
-        {/*   })} */}
-        {/* </h5> */}
-        {/* <div className={styles.switchRow}> */}
-        {/*   <LeftColumn */}
-        {/*     fieldTitle={{ id: 'field.display' }} */}
-        {/*     fieldDescription={{ id: 'description.display' }} */}
-        {/*   /> */}
-        {/*   <Switch id="display" /> */}
-        {/* </div> */}
-        {/* <span className="divider" /> */}
+        <span className="divider" />
+        <h5 className="heading-5">
+          {formatText({
+            id: 'userPreferencesPage.displayPreferences',
+          })}
+        </h5>
+        <div className={styles.switchRow}>
+          <LeftColumn
+            fieldTitle={{ id: 'field.display' }}
+            fieldDescription={{ id: 'description.display' }}
+            width=""
+          />
+          <Switch
+            id="display"
```

Comment:
Is this switch referenced by its id anywhere? If not it could be removed

Reply:
No but it's used by the input and label tags it wraps. It's also nice for debugging to be able to reference components by some other means than their tailwind classnames

Reply:
If you don't pass it explicitly one gets generated for you, hence my question. Currently it doesn't seem this id is required anywhere else